layouts/partials/example-go-grpc.html

```diff
@@ -10,50 +10,45 @@
 {{ $majorVersion := int (index $version 0) }}
 {{ $minorVersion := int (index $version 1) }}
 {{ $patchVersion := int (index $version 2) }}
-{{ $versionGe105 := or (eq $currentBranch "master") (gt $majorVersion 1) (and (eq $majorVersion 1) (gt $minorVersion 0)) (and (eq $majorVersion 1) (eq $minorVersion 0) (ge $patchVersion 5)) }}
+{{ $versionGe110 := or (eq $currentBranch "master") (gt $majorVersion 1) (and (eq $majorVersion 1) (gt $minorVersion 1)) (and (eq $majorVersion 1) (ge $minorVersion 1) (ge $patchVersion 0)) }}
```

Comment:
What versions is this PR targeting?
Code seems to say "master OR major>1 OR major=1 & minor>1 OR major=1 & minor >= 1 & patch>=0"

I think patchVersion >= 0 is always true so should be dropped.
Then second argument to OR becomes subset of the third argument, meaning we can leave only one in the code